packages/elvis/src/components/alert.scss

```diff
@@ -76,7 +76,8 @@ $global-mobile-notitle-padding: 14px; // -2px border
     width: 100%;
     max-width: unset;
     padding: $global-padding;
-    border-color: var(--e-color-signal-info);
+    border: none;
+    background-color: var(--e-color-background-element-5);
     @include breakpoints.breakpoint-up(Mobile) {
       padding: $global-mobile-padding;
     }
```

Comment:
Now that `border` is set to `none`, please ensure the padding is still correct on both desktop and mobile. I noticed that the variable `$global-mobile-padding` has a comment "`// -2px border`" (see line 10). Should this padding be updated?